variables.tf

```diff
@@ -259,6 +259,15 @@ variable "proxy_configuration" {
   default     = null
 }
 
+variable "proxy_configuration" {
```

Comment:
Ah @ethanrubio -- I take that back. Our tests caught something I did not! 

```
not ok 8 check if terraform code is valid
# (from function `setup' in test file test/.test-harness/test/terraform/validate.bats, line 7)
#   `terraform init >/dev/null' failed
# There are some problems with the configuration, described below.
# 
# The Terraform configuration must be valid before initialization so that
# Terraform can determine which modules and providers need to be installed.
# 
# Error: Duplicate variable declaration
# 
#   on variables.tf line 262:
#  262: variable "proxy_configuration" {
# 
# A variable named "proxy_configuration" was already declared at
# variables.tf:252,1-31. Variable names must be unique within a module.
# 
```

Can you update this variable name to `runtime_platform`. That should do the trick. 

Yay for tests!

Reply:
whoops copy paste fail sorry! fixed! thanks